### PR TITLE
Add support for Livewire 3

### PR DESCRIPTION
**Pull request by joshualukecaine**

### CODE REVIEW

**Comment by christophrumpel** [comment]
Hey, thanks. Can you take a look at the failing tests too?

**Comment by joshualukecaine** [comment]
need to remove php 8.0 from the test runs as it's not supported by livewire 3.0, should be 8.1 or above

**Comment by joshualukecaine** [comment]
Yeah the P8 - L9.x run needs to be removed for livewire 3

**Comment by christophrumpel** [comment]
Yeah I see, can you remove them from .github/workflows/run-tests ?

**Comment by christophrumpel** [comment]
Nice, thanks 👍 Do you know when Livewire 3 will be officially released?

**Comment by joshualukecaine** [comment]
Think it has now :)

**Comment by christophrumpel** [comment]
Indeed 😅 thanks a lot. Just one more thing, please use the new version now and not "beta".

**Comment by christophrumpel** [comment]
Hey @joshualukecaine, can you make the last change so we can release the new version?

File: composer.json [hunk 1]
```diff
@@ -17,13 +17,13 @@
...
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "spatie/laravel-package-tools": "^1.4.3",
         "illuminate/support": "^10.0|^9.0"
     },
     "require-dev": {
         "brianium/paratest": "^6.2",
-        "livewire/livewire": "^2.4",
+        "livewire/livewire": "^3.0@beta",
```
Comment (ildyria) [inline composer.json]:
```suggestion
        "livewire/livewire": "^3.0",
```
Livewire is no longer in beta.

**Comment by christophrumpel** [comment]
Thanks @joshualukecaine and @ildyria 👍

**Pull request merged**